README.md

```diff
@@ -94,11 +94,11 @@ As mentioned above, you'll need to register your client ID. We recommend you tha
 To define your app credentials, you'll need to create an instance of `Spotify::Accounts`:
 
 ```ruby
-@accounts = Spotify::Accounts.new({
+@accounts = Spotify::Accounts.new(
   client_id: "spotify client ID",
   client_secret: "spotify client secret",
   redirect_uri: "redirect URI"
-})
+)
 ```
 
 Alternatively, these credentials can be supplied as environment variables when running your application:
```

Comment:
I wasn't able to find a good source for a simple explanation of environment variables 👎